doc/user/tutorial.rst

```diff
@@ -160,15 +160,28 @@ params, as we shall see later on.
 
 Right now, the image resource responds to GET requests with a simple
 ``200 OK`` and a JSON body. Falcon's Internet media type defaults to
-``application/json`` but you can set it to whatever you like. See
-serialization with `MessagePack <http://msgpack.org/>`_ for example:
+``application/json`` but you can set it to whatever you like. For example,
+you could use `MessagePack <http://msgpack.org/>`_, or any other
+serialization format.
+
+If you'd want to use MessagePack in the above example, you would need to
```

Comment:
Just a couple minor edits for grammar:

"If you'd want to" ==> "If you'd like to"
"you would need to" ==> "you'll need to"